homeassistant/bootstrap.py

```diff
@@ -673,6 +674,10 @@ async def _async_set_up_integrations(
         requirements.async_load_installed_versions(hass, needed_requirements),
         "check installed requirements",
     )
+    hass.async_create_background_task(
```

Comment:
Maybe a comment here to explain why would be good.  We want to load the translations as soon as possible in a single job to avoid a thundering herd that all has to wait on the translations lock